react/src/lib/List/index.js

```diff
@@ -177,14 +177,14 @@ class List extends React.Component {
       case 37:
         this.getNextFocusedChild(items, tgt, -1);
         this._needsRefocus = true;
-        flag = true;
+        if(!shouldPropagateKeyDown) flag = true;
```

Comment:
Is this backwards compatible? It's hard for me to tell from the logic/case statement. We should make the default whatever it was and have the new prop change that behavior. If that is how it works then I'll approve

Reply:
+1

Reply:
@andrewholsted  - Yes, it is backwards compatible. 
[here](https://github.com/momentum-design/momentum-ui/blob/f23e9e2f6af5a7fbb785fad340f2768f6d308966/react/src/lib/List/index.js#L457), I have set the default props to `false` and in case of default value `false`, [line](https://github.com/momentum-design/momentum-ui/blob/f23e9e2f6af5a7fbb785fad340f2768f6d308966/react/src/lib/List/index.js#L187) and [line](https://github.com/momentum-design/momentum-ui/blob/f23e9e2f6af5a7fbb785fad340f2768f6d308966/react/src/lib/List/index.js#L180) would work  as previously. 
So, whenever we don't have the `shouldPropagateKeyDown` props, it should work as before.